demisto_sdk/tests/test_files/fake_pack/.pack-ignore

```diff
@@ -1,2 +1,5 @@
 [demisto-sdk]
 ignore = IN100,IN101
+
+[file:file_name]
```

Comment:
shouldn;t it be [file:<the actuall file name>]

Reply:
![image](https://user-images.githubusercontent.com/50295826/82799563-e8639980-9e82-11ea-9805-528ddb86b2e9.png)
should look like so... this case is specifically for testing